libraries/AP_Scripting/docs/docs.lua

```diff
@@ -4202,3 +4202,50 @@ function crsf:get_menu_event(events) end
 ---@param data string -- binary encoded response payload
 ---@return boolean -- true if the repsonse was successfully sent, false otherwise
 function crsf:send_write_response(data) end
+
+-- handle for DroneCAN message operations
+---@class DroneCAN_Handle_ud
+local DroneCAN_Handle_ud = {}
+
+-- create a DroneCAN_Handle, needed for all other DroneCAN message operations
+-- use close() method when finished
+---@param bus_index number -- DroneCAN bus index, 0 for first bus
+---@return DroneCAN_Handle_ud
+function DroneCAN_Handle(bus_index) end
+
+-- return true if last message was used CANFD
+---@return boolean
+function DroneCAN_Handle_ud:canfd() end
+
+-- control if next sent message on this handle should be sent as CANFD
+---@param value boolean
+function DroneCAN_Handle_ud:canfd(value) end
+
+-- set the 64 bit DroneCAN signature for the message
+---@param value uint64_t_ud
+function DroneCAN_Handle_ud:signature(value) end
+
+-- set the DroneCAN data type of the message
+---@param value integer
+function DroneCAN_Handle_ud:data_type(value) end
```

Comment:
Can't we pass these in the constructor? Would you ever change the signature and types or just get a new one?

Reply:
i think it could be in the constructor, I can't immediately think of a reason to want to change them

Reply:
changed